code/modules/clothing/under/chameleon.dm

```diff
@@ -238,6 +254,13 @@
 	resistance_flags = NONE
 	armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
 
+	species_fit = list("Vox")

```

Comment:
`drask` and `grey` may need to be in this list as well afaik

Reply:
I just copied these from their parent (i.e. gas mask), so looks like those are incorrect there as well. On further inspection, I don't think species_fit is even used anywhere. I'm just going to delete the variable entirely.

Reply:
Sweet. Yeah on second thought, looking at the procs in `clothing.dm` it seems to only really refer to `sprite_sheets` anyhow